Adjust language, grammar, and small mistakes (#19)
NETWAYS/foreman-training: architecture/03_compute_resource.md

****

Compute Resource refers to a virtualization or cloud service which is integrated into Foreman

> as a plugin. Those plugins are based on the ruby cloud service library named fog and allow

Reviewer: ```suggestion
as a plugin. Those plugins are based on the ruby cloud service library named fog and enable
```

> to create and manage virtual hosts. Some are based on a native SDK or API where no fog implementation

Reviewer: ```suggestion
creating and managing virtual hosts. Some are based on a native SDK or API where no fog implementation
```

exists or is well maintained.

Depending on the provider machines are installed from an image or unattended in the same fashion

> as a bare metal host. It allows to access the console

Reviewer: ```suggestion
as a bare metal host. It enables access to the console
```
